frontend/src/components/chats/components/select-async/select-async.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
```

Comment:
Remove this component

Reply:
I used it `const SelectAsync: React.FC<Props> = ({`